samples/system-test/v1/assetSecurityMarks.test.js

```diff
@@ -0,0 +1,95 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+'use strict';
+
+const {assert} = require('chai');
+const execa = require('execa');
+const exec = async cmd => (await execa.shell(cmd)).stdout;
+
+const organizationId = process.env['GCLOUD_ORGANIZATION'];
+
+if (organizationId) {
+  describe('client with security marks for assets', async () => {
+    let data = null;
+    before(async function() {
+      const {SecurityCenterClient} = require('@google-cloud/security-center');
+
+      // Creates a new client.
+      const client = new SecurityCenterClient();
+
+      const [assetResults] = await client.listAssets({
+        parent: client.organizationPath(organizationId),
+      });
+      const randomAsset =
+        assetResults[Math.floor(Math.random() * assetResults.length)].asset;
+      console.log('random %j', randomAsset);
```

Comment:
I had string interpolation in most places and changed it to this, because it would just print something like "random [object]".  Could you clarify your preference here (I would like to print the contents of the object, not just the fact it is an object)?

Reply:
These are objects.  @bcoe suggested eliminating, the %j and just having the object list after the string which doesn't seem to quite work either since it only seems to print top level elements properly (nested elements show up as object descriptions like '[array]')